### ISSUE TITLE
Minor issue with subsetYears

### ISSUE THREAD

**Issue by matteodefelice** [SantanderMetGroup/downscaleR#25]
In the documentation there is:
```
Logical (default is TRUE). Drop or keep dimensions of length 1
```

but there is no default.
```
> d = subsetYears(d, years = 1980:2005)
Error in asub.default(grid$Data, time.ind, grep("time", dimNames), drop = drop) : 
  argument "drop" is missing, with no default
```

Furthermore, there is a warning when executing the function:
```
> d = subsetYears(d, years = 1980:2005, drop = T)
Warning message:
In if (yrs == unique(yrs)) { :
  the condition has length > 1 and only the first element will be used
```

I'm running the last master release.

**Comment by jbedia** [comment]
Hi @matteodefelice , a couple of things:
 1. `subsetYears` is internal, thus not intended to be called by the user. You should use `subsetGrid` instead, in this case `subsetGrid(d, years = 1980:2005)` will pass `drop = TRUE` by default.
 2. The warning message you refer was fixed in commit 2ac15ddf. A will tag a new release soon
Thanks!

**Comment by jbedia** [comment]
v1.3-2 is already available. I hope it works! cheers
